src/main/java/org/mghpcc/aitelemetry/model/baremetalnode/BareMetalNodeEnUSGenApiService.java

```diff
@@ -1,48 +0,0 @@
-package org.mghpcc.aitelemetry.model.baremetalnode;
```

Comment:
@tzumainn Looks like you probably committed these changes at a moment when the code generation was recreating this file and it was blank at that moment. See if this file is back and ready to commit.

Reply:
ah yep - fixed now!